apps/site/src/app/(home)/sections/FAQ/FAQ.module.scss

```diff
@@ -0,0 +1,39 @@
+.faq {
```

Comment:
Suggestion: A lot of these properties can be replaced with Tailwind classes, except for the background-image related properties as we discussed before.